public/app/features/variables/editor/SelectionOptionsEditor.tsx

```diff
@@ -19,21 +19,21 @@ export const SelectionOptionsEditor: FunctionComponent<SelectionOptionsEditorPro
     (event: ChangeEvent<HTMLInputElement>) => {
       props.onMultiChanged(toVariableIdentifier(props.variable), event.target.checked);
     },
-    [props.onMultiChanged, props.variable]
+    [props]
   );
 
   const onIncludeAllChanged = useCallback(
     (event: ChangeEvent<HTMLInputElement>) => {
       props.onPropChange({ propName: 'includeAll', propValue: event.target.checked });
     },
-    [props.onPropChange]
+    [props]
```

Comment:
Nit: could we destruct the props in this component to prevent having dependency on the whole props object?